// nova-hyperterm/build/index.js
'use strict';

var _novaColors = require('nova-colors');

exports.decorateConfig = function (config) {
  return Object.assign({}, config, {
    cursorColor: _novaColors.uiGroups.userCurrentState,
    foregroundColor: _novaColors.uiGroups.foreground,
    backgroundColor: _novaColors.uiGroups.background,
    borderColor: _novaColors.uiGroups.background,
    colors: {
      black: _novaColors.ansiGroups.normal.black,
      red: _novaColors.ansiGroups.normal.red,
      green: _novaColors.ansiGroups.normal.green,
      yellow: _novaColors.ansiGroups.normal.yellow,
      blue: _novaColors.ansiGroups.normal.blue,
      magenta: _novaColors.ansiGroups.normal.magenta,
      cyan: _novaColors.ansiGroups.normal.cyan,
      white: _novaColors.ansiGroups.normal.white,
      lightBlack: _novaColors.ansiGroups.bright.black,
      lightRed: _novaColors.ansiGroups.bright.red,
      lightGreen: _novaColors.ansiGroups.bright.green,
      lightYellow: _novaColors.ansiGroups.bright.yellow,
      lightBlue: _novaColors.ansiGroups.bright.blue,
      lightMagenta: _novaColors.ansiGroups.bright.magenta,
      lightCyan: _novaColors.ansiGroups.bright.cyan,
      lightWhite: _novaColors.ansiGroups.bright.white
    },
    termCSS: '\n    ' + config.termCSS + ',\n    .cursor-node {\n      opacity: 0.8 !important;\n    }\n  ',
    css: '\n    ' + config.css + ',\n    .tab_first {\n      margin-left: 0 !important;\n    }\n    .tab_tab {\n      background-color: ' + _novaColors.uiGroups.gray4 + ' !important;\n    }\n    .tab_tab.tab_active {\n      background-color: ' + _novaColors.uiGroups.background + ' !important;\n      border: none !important;\n    }\n    .tab_tab.tab_active::before {\n      border: none !important;\n    }\n    .tab_text {\n      border: none !important;\n      color: ' + _novaColors.uiGroups.background + ' !important;\n    }\n    .tabs_title,\n    .tab_textActive {\n      color: ' + _novaColors.uiGroups.userCurrentState + ' !important;\n    }\n    .tabs_title,\n    .tab_textInner {\n      font-size: 14px !important;\n      font-weight: bold !important;\n    }\n    .splitpane_divider {\n      background: ' + _novaColors.uiGroups.gray2 + ' !important;\n      border-color: ' + _novaColors.uiGroups.background + ' !important;\n    }\n  '
  });
};
